packages/material-ui/src/locale/index.js

```diff
@@ -106,6 +106,27 @@ export const frFR = {
   },
 };
 
+export const idID = {
+  props: {
+    MuiTablePagination: {
+      backIconButtonText: 'Halaman sebelumnya',
+      labelRowsPerPage: 'Baris per halaman:',
+      labelDisplayedRows: ({ from, to, count }) => `${from}-${to === -1 ? count : to} of ${count}`,
```

Comment:
Any idea how we could translate this one?

Reply:
oh yeah i forgot,
it should like this..

labelDisplayedRows: ({ from, to, count }) => `${from}-${to === -1 ? count : to} dari ${count}`,

Reply:
Thanks, I have edited your comment to make it a suggestion, you can click on the "commit"  buttons now :)